src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
+$btn-color: #00acdc;
+$btn-text-color: #fff;
+$main-acent: #060b35;
+$secondary: #616070;
+$btn-hover-color: #fff;
+$card-border-color: #f3f3f3;
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__photo {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: $btn-text-color;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+
+    width: 100%;
+    height: 40px;
+    box-sizing: border-box;
+    border: 1px solid $btn-color;
+    border-radius: 5px;
+    background: $btn-color;
+
+    &:hover {
+      background: $btn-hover-color;
+      color: $btn-color;
+    }
+  }
+}
+
+.item {
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $main-acent;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    line-height: 14px;
+    text-align: left;
+    margin-bottom: 16px;
+    color: $secondary;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+    color: $secondary;
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      text-align: right;
+      color: #060b35;
+    }
+
+    &-header {
+      font-size: 12px;
+      text-align: left;
+      color: #616070;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    background: url(/src/images/star.svg);
+    height: 16px;
+    margin-right: 4px;
+    width: 16px;
+    background-repeat: no-repeat;
+  }
+
+  &.stars--4 :nth-child(-n + 4) {
+    background-image: url('/src/images/star-active.svg');
+  }
+
+  & :last-of-type {
+    margin: 0;
+  }
+}
+
+.review {
```

Comment:
there 2 BEM blocks card and stars each in it own file
<img width="836" alt="image" src="https://github.com/user-attachments/assets/c34dd476-7fa0-42ae-b490-5d9293993287">